core/server/logging/GhostLogger.js

```diff
@@ -8,6 +8,7 @@ function GhostLogger(options) {
     this.level = options.level || 'info';
     this.mode = options.mode || 'short';
     this.path = options.path || 'ghost.log';
+    this.format = options.format || 'short';
```

Comment:
can you forward this property in https://github.com/TryGhost/Ghost/blob/master/core/server/logging/index.js#L3?
`format: config.get('logging:format')`